src/pages/Home/Home.page.jsx

```diff
@@ -3,19 +3,21 @@ import React, { useRef } from 'react';
 import VideoMosaic from '../../components/VideoMosaic';
 import mockData from '../../utils/youtube-videos-mock';
 
+export function getVideosOnly() {
```

Comment:
You're exporting the function, but, you're not exporting it as well on `Home/index.js`. I suggest putting it on that file so you can use it to test it, I saw it was repeated on `VideoMosaic.test.js` file

Reply:
nice catch! I resolved it, hope I did it as it was intended 😄